apps/system/js/notifications.js

```diff
@@ -396,6 +409,10 @@ var NotificationScreen = {
       this.container.querySelector('.priority-notifications') :
       this.container.querySelector('.other-notifications');
 
+    var dir = (detail.bidi === 'ltr' ||
```

Comment:
Let's get a comment here explaining why we don't use the the `auto` value if it was passed in the notification details.